x-pack/plugins/apm/server/routes/suggestions/route.ts

```diff
@@ -8,20 +8,28 @@
 import * as t from 'io-ts';
 import { maxSuggestions } from '../../../../observability/common';
 import { getSuggestions } from '../suggestions/get_suggestions';
+import { getSuggestionsByServiceName } from '../suggestions/get_suggestions_by_service_name';
 import { getSearchAggregatedTransactions } from '../../lib/helpers/transactions';
 import { setupRequest } from '../../lib/helpers/setup_request';
 import { createApmServerRoute } from '../apm_routes/create_apm_server_route';
+import { rangeRt } from '../default_api_types';
 
 const suggestionsRoute = createApmServerRoute({
   endpoint: 'GET /internal/apm/suggestions',
-  params: t.partial({
-    query: t.type({ field: t.string, string: t.string }),
+  params: t.type({
+    query: t.intersection([
+      t.type({
+        fieldName: t.string,
+        fieldValue: t.string,
```

Comment:
Same here, I'm inclined to say `kuery` is the better option.

Reply:
Ah this makes it hard to combine it with the terms enum API, never mind this comment :)